libraries/joomla/document/api/jsonapi.php

```diff
@@ -26,7 +26,7 @@ class JDocumentApiJsonapi extends JDocumentJson implements JsonSerializable
 	 * @var    Document
 	 * @since  __DEPLOY_VERSION__
 	 */
-	protected $document;
+	public $document;
```

Comment:
Do we want to keep this protected and just have public wrapping methods for everything? Or do we want to make it public?